Sluggo/Models/AppIdentity.swift

```diff
@@ -8,11 +8,69 @@
 import Foundation
 
 class AppIdentity: Codable {
-    var authenticatedUser: UserRecord?
-    var team: TeamRecord?
-    var token: String?
-    var baseAddress: String = Constants.Config.URL_BASE
-    var pageSize = 10
+    
+    private var _authenticatedUser: UserRecord?
+    private var _team: TeamRecord?
+    private var _token: String?
+    private var _pageSize = 10
+    private var _baseAddress: String = Constants.Config.URL_BASE
+    private var _rememberMe: Bool = false
+    
+    // MARK: Computed Properties
+    var authenticatedUser: UserRecord? {
+        set(newUser) {
+            _authenticatedUser = newUser
+            enqueueSave()
+        }
+        get {
+            return _authenticatedUser
+        }
+    }
+    var team: TeamRecord? {
+        set(newTeam) {
+            _team = newTeam
+            enqueueSave()
+        }
+        get {
+            return _team
+        }
+    }
+    var token: String? {
+        set (newToken) {
+            _token = newToken
+            enqueueSave()
+        }
+        get {
+            return _token
+        }
+    }
+    var pageSize: Int {
+        set(newPageSize) {
+            _pageSize = newPageSize
+            enqueueSave()
+        }
+        get {
+            return _pageSize
+        }
+    }
+    var baseAddress: String {
+        set(newBaseAddress) {
+            _baseAddress = newBaseAddress
+            enqueueSave()
+        }
+        get {
+            return _baseAddress
+        }
+    }
+    var rememberMe: Bool {
```

Comment:
This really ought not to trigger a save, as if Remember Me is deselected, then we will not enqueue any sort of save, because we have disabled saving.

Reply:
or at the very least, by nature of the app design, we will not have the user entering this beyond the login screen

Reply:
no! I think we should persist the url differently because we could have different users login to the same device. Remember me to me sounds more like saving the current user, not the url.